src/App.vue

```diff
@@ -320,7 +321,6 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   @Action(useStore) setLastAddressChangeDate!: (x: string) => void
   @Action(useStore) setLastAnnualReportDate!: (x: string) => void
   @Action(useStore) setLastDirectorChangeDate!: (x: string) => void
-  @Action(useStore) setNameRequest!: (x: NameRequestIF) => void
```

Comment:
This is in the mixin, IDE was complaining so I removed it.

Reply:
I suggest you comment it out instead, so it's still "here" for a future developer to see.